Japanese and Spanish translation

PhABC opened the pull request:
Translate docs into spanish & Japanese
Add Frenglish github action for auto-translation

taylanpince commented on docs/pages/solutions/wallets/overview.mdx:
```diff
@@ -1,22 +1,27 @@
 # Sequence Wallets Overview
 
-Sequence offers multiple wallet options to integrate into your products. We have two main choices: the Embedded Wallet and the Universal Wallet. Both are completely non-custodial.
+Sequence provides tools for developers to build games and blockchain apps that users love. We focus on smooth gaming, advanced web3 functions, and compatibility with all blockchain accounts. We offer three main tools: The Embedded Wallet, the Universal Wallet, and SequenceKit. These integrate with popular game engines like Unreal and Unity; with established blockchain wallets like MetaMask; and a ...
```
This is incorrect. We offer two wallet solutions: Embedded Wallet and Universal Wallet. Sequence Kit is an SDK that can be used to integrate either solution. We are trying to change language across the board to move away from this exact confusion :)

taylanpince commented on docs/pages/solutions/wallets/overview.mdx:
```diff
@@ -1,22 +1,27 @@
-On the other hand, the Universal Wallet targets native blockchain users, particularly those involved in gaming and gaming assets. It provides a familiar environment for existing blockchain users and enables developers to create compelling web3 experiences. The [Typescript SDK](/sdk/typescript/guides/overview) makes integration straightforward.
+- Game developers use the Embedded Wallet to give players immediate game access with accounts on demand, and logins via social platforms, phone, or email.  Your users can enjoy games without worrying about the blockchain.
+- Blockchain application developers use the Universal Wallet for more advanced controls and out of game experiences for their users across the Ethereum ecosystem.
+- SequenceKit [SequenceKit](/sdk/sequence-kit/overview) combines the features of both the Embedded and Universal Wallets into a customizable UI. You can attract a broader user base by accommodating users with pre-existing wallets and those who prefer the convenience of social logins. The [Typescript SDK](/sdk/typescript/guides/overview) makes integration straightforward.
```
Wherever Sequence Kit is mentioned, all other SDKs should be mentioned. Sequence Kit is a web SDK only and should be grouped with sequence.js (programmatic TypeScript SDK), Unity SDK and Unreal SDK.

taylanpince reviewed (changes requested):
Some adjustments needed.

viv-cheung committed:
Review: Initial translations for Japanese and Spanish

viv-cheung committed:
update package

viv-cheung committed:
update package

viv-cheung committed:
update package

viv-cheung committed:
detect branch

viv-cheung committed:
Integrate frenglish

Complete initial translations for ja and es

Added: github action and github scripts

PhABC committed:
Disable Frenglish github action for now

viv-cheung committed:
update home page

viv-cheung committed:
Update github action

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: frenglish-translation

viv-cheung committed:
update home pages

github-actions[bot] committed:
chore(i18n): update translations [en]

Sync file structure, format locales. Branch: frenglish-translation
